server/js/sockets.js

```diff
@@ -374,13 +374,13 @@ sockets.init = function socketInit(server) {
 
     socket.on('start-das-recording', () => {
       [1, 2, 3, 4].forEach((n) =>
-        mqttClient.publish(WirelessModule.id(n).start),
+        mqttClient.publish(WirelessModule.id(n).start, 'true'),
```

Comment:
Totally get why you added this message! Here's an alternative though. Use the `-v` (verbose) option for `mosquitto_sub`, which will print the topic as well as the message whenever one is received. This way, you'll still see empty messages!

Personally I think we could remove the `true` message from this and where it already existed above, but not a big deal. Whichever you do, can you make sure our documentation is up to date? https://www.notion.so/monashhumanpower/V3-MQTT-Topics-66e6715d0e1941ffaa82020e5868fbae#3e9b777757d443ec854feeb75c9b44d5

Reply:
oooh love that tip cheers!!!
and sure I'll fix up the docs :)